### ISSUE TITLE
`yarn install --frozen-lockfile` not working; cannot find module 'clean'

### ISSUE THREAD

**Issue by AlexSWall** [hyhugh/coc-erlang_ls#12]
When I `git clone` the repository and then run `yarn install --frozen-lockfile`, I'm getting the following error:
```
$ yarn install --frozen-lockfile
yarn install v1.22.10
[1/5] 🔍  Validating package.json...
warning coc-erlang_ls@0.0.1: The engine "coc" appears to be invalid.
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
[5/5] 🔨  Building fresh packages...
$ npx npm-run-all clean build
Watching /Users/alex/.vim/plug-plugins/coc-erlang_ls and all sub-directories not excluded by your .gitignore. Will not monitor dotfiles.
Found & ignored ./.git/logs ; is listed in .gitignore
Found & ignored ./node_modules ; is listed in .gitignore

Starting: clean
node:internal/modules/cjs/loader:930
  throw err;
  ^

Error: Cannot find module '/Users/alex/.vim/plug-plugins/coc-erlang_ls/clean'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:927:15)
    at Function.Module._load (node:internal/modules/cjs/loader:772:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```
Indeed, there is no such file 'clean' included in the repository.

Any ideas what the fix is?

**Comment by hyhugh** [comment]
Hi Alex, 
TL;DR, you shouldn't need this plugin to use erlang_ls anymore. Please refer to : https://erlang-ls.github.io/getting-started/vim/ for the instructions

Long version:
`erlang_ls` used to only support TCP client-server communication long long time ago, and this plugin was supposed to establish/re-connect TCP to erlang_ls. But now(long time ago actually), erlang_ls added support for studio client-server communication (which is supported by coc by default), you should be able to use erlang_ls with coc.nvim without this plugin.
